examples/wiki/wiki.py

```diff
@@ -9,6 +9,10 @@
 WIKIPART = re.compile(r'([A-Z][a-z0-9_]+)')
 WIKIWORD = re.compile(r'([A-Z][a-z0-9_]+(?:[A-Z][a-z0-9_]+)+)')
 
+@app.route('/', methods=['GET'])
+def redirect():
+    return redirect(url_for('HomePage'))
```

Comment:
I don't think `url_for` works this way. I think you have to give it the name of an endpoint (`show_page` in this case) with whatever arguments that endpoint accepts. You probably want something like `url_for("show_page", page="HomePage"` -- but please test to be sure I'm not making this up.